### PR TITLE
Need to install crcmod for GCP cli based tests.

### PR DESCRIPTION
**Pull request by stfeng**

### CODE REVIEW

**Comment by stfeng** [comment]
@yuyantingzero Please take a look, thanks!

File: perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py [hunk 1]
```diff
@@ -752,6 +752,7 @@ def Prepare(self, vm):
                      '--rc-path=.bash_profile '
                      '--path-update=true '
                      '--bash-completion=true')
+    vm.RemoteCommand('sudo pip install -U crcmod')
```
Comment (yuyantingzero) [inline perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py]: Please add a help message here "gsutil help crc32c".  Otherwise LGTM

Comment (stfeng) [inline perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py]: Done. Merging now...

File: perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py [hunk 2]
```diff
@@ -752,6 +752,8 @@ def Prepare(self, vm):
                      '--rc-path=.bash_profile '
                      '--path-update=true '
                      '--bash-completion=true')
+    # Need to install crcmod for gcp. See "gsutil help crc" for details.
+    vm.RemoteCommand('sudo pip install -U crcmod')
```
Comment (voellm) [inline perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py]: How about checking to see if it is installed first and logging.  We should also differentiate the default experience from the improved experience by adding meta data on when you have to stall crcmod.

How do you plan to do that?

Comment (stfeng) [inline perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py]: Yes we should be able to add a metadata entry to indicate if the crcmod is installed by default or by the benchmark. I filed issue 457 and assigned to myself. Will get to that soon.

Comment (voellm) [inline perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py]: I'd prefer to do it as part of this checkin vs a new issue.

On Tue, Sep 8, 2015 at 3:56 PM, stfeng <notifications@github.com> wrote:

> In perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py
> <https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/456#discussion_r38990464>
> :
>
> > @@ -752,6 +752,8 @@ def Prepare(self, vm):
> >                       '--rc-path=.bash_profile '
> >                       '--path-update=true '
> >                       '--bash-completion=true')
> > +    # Need to install crcmod for gcp. See "gsutil help crc" for details.
> > +    vm.RemoteCommand('sudo pip install -U crcmod')
>
> Yes we should be able to add a metadata entry to indicate if the crcmod is
> installed by default or by the benchmark. I filed issue 457 and assigned to
> myself. Will get to that soon.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/456/files#r38990464>
> .
>



-- 
Anthony F. Voellm (aka Tony)
Google Voice:  (650) 516-7382
<https://www.google.com/voice/b/0?pli=1#phones>
Blog: http://perfguy.blogspot.com/
Benchmark like a pro... PerfKitBenchmarker
<https://github.com/GoogleCloudPlatform/PerfKitBenchmarker>

**Pull request merged**